index.html

```diff
@@ -0,0 +1,285 @@
+    
+
+      <!--PLACE HOLDER IMAGE2-->
+      <div class="place-holder-img">
+        <img src="./img/ImgPlaceholder.png" alt="place holder">
+       </div>
+
+
+      <!-- MUTLI POST STORIES -->
+      <div class="title-post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="paragraph2">
+        <p>A daily selection of privately <br>
+           personalized reads; no accounts <br>
+           or sign-ups required. has been the <br>
+           industry's standard dummy text <br>
+           ever since the 1500s, when an <br>
+           unknown printer took a standard <br>
+           dummy text.</p>
+      </div>
+
+      <!-- languajes de programacion -->
+      <ul class="programming-languages">
+        <li><a href="">CSS</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">html</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">bootstrap</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">Ruby</a></li> 
+      </ul>
+
+      <!-- SEE PROJECT BUTTON -->
+      <div class="see-project">
+        <img src="./img/Enable.png"alt="see project"></div>
+
+
+    <!--PLACE HOLDER IMAGE2-->
+  <div class="place-holder-img">
+ <img src="./img/ImgPlaceholder.png" alt="place holder">
+</div>
+<!-- MUTLI POST STORIES -->
+<div class="title-post">
+  <h2>Multi-Post Stories</h2>
+</div>
+<div class="paragraph2">
+  <p>A daily selection of privately <br>
+     personalized reads; no accounts <br>
+     or sign-ups required. has been the <br>
+     industry's standard dummy text <br>
+     ever since the 1500s, when an <br>
+     unknown printer took a standard <br>
+     dummy text.</p>
+</div>
+
+  <!-- languajes de programacion -->
+  <ul class="programming-languages">
+  <li><a href="">CSS</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">html</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">bootstrap</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">Ruby</a></li> 
+</ul>
+
+<!-- SEE PROJECT BUTTON -->
+<div class="see-project">
+<img src="./img/Enable.png"alt="see project"></div>
+
+<div class="contact-us">
+  <h2>CONTACT US</h2>
+</div>
+
+<!-- paragraph 3 -->
+<div class="paragraph3">
+  <p>If you have an application you are <br>
+     interested in developing, a feature that you <br>
+     need built or a project that needs coding. <br>
+     I’d love to help with it </p>
+</div>
+  <ul class="buttom">
+    <li class="input-text">
+      input-text
+    </li>
+  </ul>
+
+  <ul class="buttom">
+    <li class="input-email">
+      jeremy@gmail.com
+    </li>
+  </ul>
+  <ul class="buttom">
+    <li class="input-comment">
+      input-comment
+    </li>
+  </ul>
```

Comment:
- [ ] Good job implementing this in your project 💯, but kindly remove this part since you will work on it in a future project
- [ ] Kindly add the About Me section since it is a mandatory milestone for this project